src/examples/component-definition/json/example-component.json

```diff
@@ -40,7 +40,7 @@
                     },
                     {
                         "uuid": "22dbff65-9729-449f-9dfc-4e5fee0906de",
-                        "source": "../../../content/fedramp.gov/json/FedRAMP_HIGH-baseline_profile.json",
+                        "source": "https://raw.githubusercontent.com/GSA/fedramp-automation/master/baselines/rev4/json/FedRAMP_rev4_HIGH-baseline_profile.json",
```

Comment:
@ohsh6o Is this the best (most stable and authoritative) version of this file to reference?

Reply:
Ditto here, re the most stable and authoritative. How do you feel about this and the other one set to `master` and changing to a version tag or branch accordingly once they are created?